kong/templates/nginx_kong.lua

```diff
@@ -246,8 +254,9 @@ server {
 
     ssl_session_cache shared:SSL:10m;
     ssl_session_timeout 10m;
```

Comment:
One small suggestion is to bump this to `1d` (as suggested by Mozilla).

The session ticket cache has LRUs and will keep the newest one in.

Reply:
@dndx I added this as part of the suite